# Define the goal
# string name
---
# Define the result
bool success
grasp_demo/BoundingBox targetBB
---
# Define the feedback
bool in_progress
grasp_demo/BoundingBoxes detectedBB
